data/mods/No_Filthy_CBMs/modinfo.json

```
@@ -0,0 +1,10 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "no_filthy_cbm",
+    "name": "No Filthy CBM",
```

I guess this short name could be slightly misleading for some individuals, especially if they don't read the description.
```suggestion
    "name": "No Filthy CBMs From Zombies",
```